packages/core-components/src/components/DependencyGraph/DependencyGraph.test.tsx

```diff
@@ -49,6 +49,28 @@ describe('<DependencyGraph />', () => {
     expect(queryAllByTestId(LABEL_TEST_ID)).toHaveLength(0);
   });
 
+  it('update render if already referenced nodes are added later', async () => {
+    const { getByText, queryAllByTestId, findAllByTestId, rerender } = render(
```

Comment:
Personal preference, I've enjoyed using `import { screen } from '@testing-library/react'` and doing like `expect(screen.getByText...` to make these things more terse